pkg/converter/sloppy_file.go

```diff
@@ -172,6 +188,35 @@ func (sf *SloppyFile) convertCommand(cmd []interface{}) *string {
 	return &str
 }
 
+var resourceMemRegex = regexp.MustCompile(`^(\d+)([bkmgBKMG]){1}$`)
+
+func (sf *SloppyFile) convertMemoryResource(res string) (*int, error) {
+	const lowestMem float64 = 64 // lowest mem size sloppy.io supports
+	match := resourceMemRegex.FindStringSubmatch(res)
+	if len(match) == 3 {
+		memResource, err := strconv.Atoi(match[1])
+		if err != nil {
+			return nil, err
+		}
+		unit := strings.ToLower(match[2])
+		var mem int
+		switch unit {
+		case "b":
+			mem = int(math.Max(float64(memResource/1024/1024), lowestMem))
+		case "k":
+			mem = int(math.Max(float64(memResource/1024), lowestMem))
+		case "m":
+			mem = memResource
+		case "g":
+			mem = memResource * 1024
+		default:
+			return nil, fmt.Errorf("convert resources: unsupported memory format: %q", res)
```

Comment:
From this error message, its not clear that the value is coming from `Deploy.Resources.Limits.Memory` anymore (or `deploy.resources.limits.memory` in json/yml casing). Should include that in some form.